daemon/create.go

```diff
@@ -310,19 +310,11 @@ func (daemon *Daemon) mergeAndVerifyConfig(config *containertypes.Config, img *i
 	return nil
 }
 
-// Checks if the client set configurations for more than one network while creating a container
-// Also checks if the IPAMConfig is valid
+// Checks if the IPAMConfig is valid
 func verifyNetworkingConfig(nwConfig *networktypes.NetworkingConfig) error {
```

Comment:
While most of the validation is for IPAMConfig, it's still validating the networking config as a whole, so perhaps something like;


```suggestion
// verifyNetworkingConfig  validates if the nwConfig is valid.
func verifyNetworkingConfig(nwConfig *networktypes.NetworkingConfig) error {
```


That said, I wonder if (for a follow-up), the IPAM validation should be extracted.